ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/CSharpOutputCustomizer.java

```diff
@@ -0,0 +1,187 @@
+/*******************************************************************************
+ * Copyright (c) 2017 RedHat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   RedHat, Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.console;
+
+import static com.google.gwt.regexp.shared.RegExp.compile;
+
+import org.eclipse.che.ide.api.app.AppContext;
+import org.eclipse.che.ide.api.editor.EditorAgent;
+import org.eclipse.che.ide.resource.Path;
+
+import com.google.gwt.regexp.shared.RegExp;
+
+/**
+ * Output customizer adds an anchor link to the lines that match a .NET C#
+ * compilation error or warning message and a stack trace line pattern and
+ * installs the handler functions for the links. The handler parses the stack
+ * trace line, searches for a candidate C# file to navigate to, opens the found
+ * file in editor and reveals it to the required line and column (if available)
+ * according to the line information
```

Comment:
Next time is OK?
Or I should add it to all the output customizer like classes (as well as to those which was added only in fix for CHE-15)?